lib/base/application.cpp

```diff
@@ -746,11 +754,14 @@ void Application::SigAbrtHandler(int)
 		Log(LogCritical, "Application")
 			<< "Icinga 2 has terminated unexpectedly. Additional information can be found in '" << fname << "'" << "\n";
 
+		ofs << "Caught SIGABRT." << "\n"
+			<< "Current time: " << Utility::FormatDateTime("%Y-%m-%d %H:%M:%S %z", Utility::GetTime()) << "\n"
+			<< "\n";
+
 		DisplayInfoMessage(ofs);
 
-		StackTrace trace;
-		ofs << "Stacktrace:" << "\n";
-		trace.Print(ofs, 1);
+		ofs << "\n" << "Stacktrace:" << "\n"
```

Comment:
There are three \\n between the time and "Stacktrace:". Intended?

Reply:
`DisplayInfoMessage` also prints to `ofs`. So in the output there are just multiple sections sections separated by an empty line.

Reply:
OK.